src/System.Management.Automation/engine/LanguagePrimitives.cs

```diff
@@ -591,9 +591,7 @@ public static PSDataCollection<PSObject> GetPSDataCollection(object inputValue)
         /// <param name="second">Object to compare first to.</param>
```

Comment:
@vexx32, your last commit had 2 failures in `PowerShell-CI-macos`
SemanticVersion api tests.Error handling.Missing parts: 'NullString': '$null'
```powershell
Expected an exception, with FullyQualifiedErrorId 'FormatException' to be thrown, but the FullyQualifiedErrorId was 'PSArgumentNullException'. from /Users/vsts/agent/2.153.2/work/1/s/test/powershell/engine/Basic/SemanticVersion.Tests.ps1:209 char:19
    +                 { [SemanticVersion]::Parse($version) } | Should -Thro ...
    +                   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
at <ScriptBlock>, /Users/vsts/agent/2.153.2/work/1/s/test/powershell/engine/Basic/SemanticVersion.Tests.ps1: line 209
209:                 { [SemanticVersion]::Parse($version) } | Should -Throw -ErrorId "FormatException"
```
Null Representatives.Casting Behaviour.Error occurred in Context block
```powershell
Unable to find type [System.Management.Automation.AutomationNull].
At /Users/vsts/agent/2.153.2/work/1/s/test/powershell/Language/Scripting/NullRepresentatives.Tests.ps1:51 char:28
```